### ISSUE TITLE
New messages are not being sent and received after initial history loading

### ISSUE THREAD

**Issue by moorchegue** [wee-slack/wee-slack#643]
I haven't found any similar complaints here, so I'm guessing it's a rare case. wee-slack successfully connects to the server, loads channel history, but fails to receive any new messsages. When I'm trying to send a message I get errors like this:
```
13:32:55            weechat | python: stdout/stderr (slack): WS ERROR
```

The behavior is consistent when obtaining a token with /slack register or using a legacy token.

Any ideas what might be wrong and how to debug this further?

Versions: wee-slack 2.1.1-1 (and since at least 2.0.0), weechat 2.2-2.

**Comment by moorchegue** [comment]
I found what was wrong in my case. I was using proxychains to quickly bypass the firewall that blocks Slack. I didn't think that would be a problem since initial connection was established and channels list, messages, etc got fetched no problem. But apparently at some point this method fails resulting in ^^^

Once I switched to /proxy (which wee-slack totally supports through network.curl_proxy) everything started working smoothly.

@xcko, I wonder if your problem is somewhat similar. If it is, maybe this issue should be closed.
